### ISSUE TITLE
failed to find element matching selector "[aria-label="Home"]

### ISSUE THREAD

**Issue by castaway** [carpii/barclayscrape#3]
```
Selecting account 20845883789160
Fetched OFX for account 20845883789160
node:internal/process/promises:288
            triggerUncaughtException(err, true /* fromPromise */);
            ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "Error when clicking [aria-label="Home"] on URL https://bank.barclays.co.uk/olb/balances/PersonalFinancialSummary.action#/transactions: Error: Error: failed to find element matching selector "[aria-label="Home"]"".] {
  code: 'ERR_UNHANDLED_REJECTION'
} 
```

New one today, not sure if its an ongoing one, presumably they renamed something (again)?

**Comment by carpii** [comment]
Whats the full commandline are you using (excluding anything sensitive)?

Is there any further debug output if you run using `node --trace-warnings`

**Comment by carpii** [comment]
Looks like Barclays have done a general redesign of the top menu, affecting most navigation.

Latest push fixes the `get_combined_ofx` feature which was also broken

I'll tackle `get_ofx` as soon as I can

**Comment by castaway** [comment]
Ah didnt see the first notification, good you're on the case `"$ENV{BARCLAYSCRAPE}/barclayscrape.js --otp $ENV{CODE} get_ofx $ENV{ACCESS_HOME}"`

**Comment by carpii** [comment]
For `get_ofx`, could you describe the exact steps and clicks you'd take, if you were trying to do this manually from the website?

The current code works by navigating to some direct url's, and I think some of these are now defunct

**Comment by castaway** [comment]
After login the account name click gets me to this page (as above):
* https://bank.barclays.co.uk/olb/balances/PersonalFinancialSummary.action#/transactions
At the bottom of this page there's the "Export All" drop down, which exposes a list of links, among them this one:
```
<li data-ng-repeat="package in exportList" class="ng-scope"> <a data-ng-click="configureForExportTransaction(package.packageCode,package.name)" href="#" class="my-account-link last-export-item" data-ng-class="{'last-export-item': $index == exportList.length-1}" data-ng-blur="closeExportOnBlur($event)"> <span data-ng-bind-html="package.list" class="ng-binding">OFX(Microsoft ® Money 98/Microsoft ® Money 99/Microsoft ® Money 2000/Microsoft ® Money 2001/Microsoft ® Money 2002/Sage Line 50)</span> </a> </li>```

**Comment by carpii** [comment]
Could you try latest commit? I think this ought to fix `get_ofx`

**Comment by castaway** [comment]
Yup looks good!
